static/app/components/calendar/calendarStylesWrapper.tsx

```diff
@@ -1,13 +1,22 @@
 import 'react-date-range/dist/styles.css';
 import 'react-date-range/dist/theme/default.css';
 
-import {DateRangePicker as BaseDateRangePicker} from 'react-date-range';
 import styled from '@emotion/styled';
 
 import space from 'sentry/styles/space';
 
-const DateRangePicker = styled(BaseDateRangePicker)`
-  padding: 21px; /* this is specifically so we can align borders */
+const CalendarStylesWrapper = styled('div')`
+  padding: ${space(3)};
+
+  .rdrCalendarWrapper:not(.rdrDateRangeWrapper) {
+    .rdrDayHovered {
+      .rdrDayNumber {
+        &:after {
+          border: 0;
+        }
+      }
+    }
+  }
```

Comment:
Any reason not to slam this all together?

```suggestion
  .rdrCalendarWrapper:not(.rdrDateRangeWrapper) .rdrDayHovered .rdrDayNumber:after {
    border: 0;
  }
```

Reply:
🪆